fluxy/fluxy.py

```diff
@@ -455,3 +475,11 @@ def last() -> Last:
 
 def mean(column: str) -> Mean:
     return Mean(column)
+
+
+def limit(n: int, offset: int = 0):
+    return Limit(n, offset)
+
+
+def sort(columns: list[str] = ["_value"], desc: bool = False):
+    return Sort(columns, desc)
```

Comment:
To me having the `desc` be allowed as a positional argument can be a bit tricky. I think it'd be most pythony to use
```suggestion
def sort(columns: list[str] = ["_value"], *_args, desc: bool = False):
    return Sort(columns, desc)
```
to force the named argument use: `sort(["_value"], desc=True)`.

Or (more Java) introduce an `Order` enum so you'd have `sort(["_value"], Order.DESC)`

Reply:
I chose these 2 positional arguments because they are the only 2 relevant ones you can use: https://docs.influxdata.com/flux/v0/stdlib/universe/sort/. So not sure if we need the `*_args`?

Reply:
Flux has named arguments, so it's clear that `desc: true` isn't `asc: true`, whereas a person reading `sort(["_value"], True)` might think that the `True` might mean ascending instead of descending. Come to think of it, having ascending be `True` would be more natural to me. It really helps to force people to specify what it is that's being `True`

Reply:
Okay, fixed it with an enum, let me know what you think!